src/Microsoft.ML.Transforms/ExtensionsCatalog.cs

```diff
@@ -5,39 +5,44 @@
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Transforms;
-using Microsoft.ML.Transforms.Conversions;
 
 namespace Microsoft.ML
 {
-    public static class MissingValueIndicatorCatalog
+    public static class ExtensionsCatalog
     {
         /// <summary>
-        /// Initializes a new instance of <see cref="MissingValueIndicatorEstimator"/>
+        /// Creates a new output column, of boolean type, with the same number of slots as the input column. The value in the output column
+        /// is true if the value in the input column is missing.
         /// </summary>
-        /// <param name="catalog">The categorical transform's catalog.</param>
+        /// <param name="catalog">The transform extensions' catalog.</param>
```

Comment:
>' [](start = 58, length = 1)

sneaky apostrophe is sneaky. 
across whole file.

Reply:
This is fine, no? The extensions are transform extensions; there are many of them. Possession on plural is followed by apos. 

---
In reply to: [236449673](https://github.com/dotnet/machinelearning/pull/1682#discussion_r236449673) [](ancestors = 236449673)